sktime/transformations/series/exponent.py

```diff
@@ -189,7 +189,7 @@ def get_test_params(cls, parameter_set="default"):
 
 
 class SqrtTransformer(ExponentTransformer):
-    """Apply element-sise square root transformation to a time series.
+    """Apply element-size square root transformation to a time series.
```

Comment:
```suggestion
    """Apply elementwise square root transformation to a time series.
```

This makes more sense in my opinion.

Reply:
Done.